.github/workflows/rust.yml

```diff
@@ -165,6 +165,9 @@ jobs:
         run: |
           forest-cli snapshot export
           sha256sum -c *.sha256sum
+      - name: validate snapshot
+        run: |
+          forest-cli --chain calibnet snapshot validate $SNAPSHOT_DIRECTORY/*.car --recent-stateroots 2000 --force
```

Comment:
Maybe without the `--recent-stateroots` argument since we also omit it when exporting a snapshot. Also, let's check that validating the snapshot as a mainnet snapshot fails.